app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/exemptionandlosses/ExemptionAndLossesController.scala

```diff
@@ -269,13 +288,23 @@ class ExemptionAndLossesController @Inject() (
                 c => inYearLossesForm.fill(c.inYearLosses.inPounds())
               )
             )(
-              page = inYearLossesPage(
-                _,
-                _,
-                disposalDate,
-                fillingOutReturn.subscribedDetails.isATrust,
-                draftReturn.representativeType()
-              )
+              page =
+                if (fillingOutReturn.isFurtherReturn.contains(true))
+                  furtherReturnInYearLossesPage(
+                    _,
+                    _,
+                    fillingOutReturn.subscribedDetails.isATrust,
+                    draftReturn.representativeType(),
+                    fillingOutReturn.isFurtherReturn
+                  )
+                else
```

Comment:
it will hit the `else` when it is `None` - is that ok?

Reply:
yes, if its not true, means not a further return then we use existing page.